src/renderer/components/Wallet/WalletSidebar/WalletSidebarFilters.vue

```diff
@@ -155,7 +142,19 @@ export default {
       },
       sort: {
         order: this.sortOrder
-      }
+      },
+      sortOptions: [
```

Comment:
@dated since this `Array` shouldn't change, it could be moved, frozen, to `$options`.

Reply:
done